### PR TITLE
Fixes #5663

### LINKED ISSUE #5663: Mommis picking up paper
Its not a serious issue to me, but its kinda of annoying. As a mommi, when you click to pick up a sheet of paper, it opens the paper so you can read it. In order to pick up the paper you have to right click it. Like I said, this isnt really important, but kinda annoying

### PR DESCRIPTION
**Pull request by clusterfack**
Mommis can now pick up paper

### CODE REVIEW

File: code/modules/paperwork/paper.dm [hunk 1]
```diff
@@ -87,6 +87,12 @@
 				spam_flag = 0
 	return
 
+/obj/item/weapon/paper/attack_robot(var/mob/user as mob)
+	if(istype(user,/mob/living/silicon/robot/mommi) && Adjacent(user))
```
Comment (PJB3005) [inline code/modules/paperwork/paper.dm]: `ismommi()`

File: code/modules/paperwork/paper.dm [hunk 2]
```diff
@@ -87,6 +87,12 @@
...
+		attack_hand(user)
```
Comment (PJB3005) [inline code/modules/paperwork/paper.dm]: `return attack_hand(user)`

File: code/modules/paperwork/paper.dm [hunk 3]
```diff
@@ -87,6 +87,12 @@
...
+	else
+		attack_ai(user)
```
Comment (PJB3005) [inline code/modules/paperwork/paper.dm]: `return attack_ai(user)`

**Comment by clusterfack** [comment]
*Stuff that is literally the same*

**Comment by PJB3005** [comment]
\>implying

**Comment by N3X15** [comment]
Needs fixing

**Comment by PJB3005** [comment]
\>Merged it before travis completed, didn't compile, didn't test it.

**Comment by clusterfack** [comment]
*corrects someone using incorrect information*

**Comment by PJB3005** [comment]
\>Listening to people's advice without checking it.

**Comment by clusterfack** [comment]
But I think we can all say for sure that this was PJB's fault for asking for an unnecessary change that didn't even work

**Comment by PJB3005** [comment]
And we can all say it's @clusterfack's fault for merging something untested before travis completed.

**Comment by clusterfack** [comment]
Travis is for scrubs, pjb you're a scrub

**Pull request merged**